spec/lib/msf/base/serializer/readable_text_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+require 'msf/base/serializer/readable_text'
+# require 'rex/post/meterpreter/extensions/stdapi/net/interface'
+# require 'rex/post/meterpreter/extensions/stdapi/net/route'
+
+describe Msf::Serializer::ReadableText do
+
+  let (:subject) {
```

Comment:
`subject` has a special meaning for RSpec; putting it in a `let` is weird.  In general, you should have a descriptive name for your subject, e.g.:
```ruby
subject(:serializer) { described_class }
```
and then in your `specify` or `it` blocks, you would use `serializer`.